src/DIPS.Xamarin.UI/Resources/Colors/ColorPalette.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Resources.Colors
+{
+#pragma warning disable 1591
+    public static class ColorPalette
+    {
+        public static Color Dark = Color.FromHex("#000000");

```

Comment:
Disable comment inside and explain what this class of colors is meant for?

Reply:
Move pragma disable down to property level and explain class? Sure thing, I will have to ask for an description from our designers